test/integration/included/stanc.expected

```diff
@@ -286,9 +283,9 @@ Invalid character found.
 
   $ ../../../../install/default/bin/stanc --include_paths="../included" --auto-format stanc_helper_with_bad_include_parse_error.stan
 
-Syntax error in file ../included/parse-err.stan, line 2, columns 2-9, included from
-file ../included/incl_stanc_helper_parse_error.stan, line 2, column 2, included from
-file stanc_helper_with_bad_include_parse_error.stan, line 2, column 0, parsing error:
+Syntax error in '../included/parse-err.stan', line 2, column 2, included from
+'../included/incl_stanc_helper_parse_error.stan', line 2, column 2, included from
+'stanc_helper_with_bad_include_parse_error.stan', line 2, column 0 to column 9, parsing error:
```

Comment:
This doesn't seem quite right.

I'm sorry by the way, I know this code is horrible with all the string mangling.

Reply:
I'm tempted to just leave off the 2nd location entirely if there are includes - it's super difficult as a user of this to parse the hierarchical include structure when two locations are involved. Would that work for you?

Reply:
Yes, I think that's probably the easiest way out here. When we have includes it's super hard to understand these error messages anyway.